registry/file.go

```diff
@@ -39,6 +39,12 @@ func (r *fileRegistry) ListClusters(filter Filter) ([]*api.Cluster, error) {
 		return nil, err
 	}
 
+	for _, cluster := range fileClusters.Clusters {
+		for _, nodePool := range cluster.NodePools {
+			nodePool.InstanceType = nodePool.InstanceTypes[0]
```

Comment:
Do we want to check that `len(InstanceTypes) > 0`?

Reply:
Fixed.